target/linux/ramips/dts/mt7621_ruijie_rg-ew1200g-pro-v1.1.dts

```diff
@@ -48,12 +48,12 @@
 	flash@0 {
 		compatible = "jedec,spi-nor";
 		reg = <0>;
-		#address-cells = <1>;
-		#size-cells = <1>;
```

Comment:
Adding these lines removes the warning:
"OF: Bad cell count for /palmbus@10000000/spi@b00/flash@0/partitions"

If you delete them the warning will come back.

Reply:
This warning is an upstream bug. The child node `partition` doesn't have `reg` property, we should not add `#address-cells` and `#size-cells` here.

Reply:
How else can we fix this bug?

Reply:
It's harmless. If you are interested, you can try tracking the source code.

Reply:
I'd rather have some pointless lines (dtc doesn't complain about them, right?) than a user-visible error/warning on every boot.

Once this is fixed upstream, we can remove them; but until then I'd rather keep them in.

Reply:
> I'd rather have some pointless lines (dtc doesn't complain about them, right?) than a user-visible error/warning on every boot.

Though dtc did't complain about it, it seems that the driver has done some address translation operations. I'm not sure if it's really harmless. ref: https://elixir.bootlin.com/linux/v6.6.82/source/drivers/of/address.c#L592

If it's really annoying, I'd prefer just silence the printf(): https://github.com/openwrt/openwrt/pull/11302/commits/e74219c795a03e8b9adccbb5e6a20d4566872f8a

> 
> Once this is fixed upstream, we can remove them; but until then I'd rather keep them in.

Anyway, I'll revert this change.